components/Ui/UiCard.vue

```diff
@@ -86,17 +83,12 @@
 </template>
 
 <script setup lang="ts">
-// import "@carbon/web-components/es/components/tag/tag.js";
-// import "@carbon/web-components/es/components/tooltip/tooltip-icon.js";
-import Information16 from "@carbon/icons-vue/lib/information/16";
 import { Link } from "~/types/links";
 import { CtaClickedEventProp } from "~/types/segment";
 
-export interface TagTooltip {
+export interface PrimaryTag {
```

Comment:
We don't need this interface since the primary tag is a string

Reply:
Removed with https://github.com/Qiskit/qiskit.org/pull/3322/commits/033418cf80dca3b0a41e7aaeccb0cc63685ed2bc